snaps/features/custom-ui/index.md

```diff
@@ -369,6 +418,55 @@ const interfaceId = await snap.request({
 <img src={require("../../assets/custom-ui-checkbox.png").default} alt="Checkbox UI example" width="450px" style={{border: "1px solid #DCDCDC"}} />
 </p>
 
+### `Container`
+
+Outputs a container component that can hold a box of content and an optional footer. This is useful for creating structured layouts with action buttons at the bottom in [custom dialogs](dialogs.md#display-a-custom-dialog).
+
+#### Props
+
+- `backgroundColor`: `"default" | "alternative"` - (optional) The background color of the container. Defaults to `"default"`.
+- `children`: Can be either:
+  - A single Box element, or
+  - An array containing a Box element and a Footer element
```

Comment:
```suggestion
- `backgroundColor` - (Optional) The background color of the container.
  Possible values are `"default"` and `"alternative"`.
  The default is `"default"`.
- `children`: The contents of the container.
  This can be a single [`Box`](#box) component, or an array containing a [`Box`](#box) component and a [`Footer`](#footer) component.
```